src/platforms/common/performance/index.mdx

```diff
@@ -12,13 +12,20 @@ supported:
   - go
   - android
   - ruby
+  - apple
 notSupported:
   - javascript.cordova
 description: "Learn how to enable performance monitoring in your app if it is not already set up."
 redirect_from:
   - /platforms/javascript/performance/apm-to-tracing/
 ---
 
+<PlatformSection supported={["apple"]}>
+<Note>
+Attention: This is a preview API available since 7.0.0-alpha.2. This API is not in stable state yet. It may change in the future.
+</Note>
+</PlatformSection>
```

Comment:
```suggestion
<PlatformSection supported={["apple"]}>

<Note>

Attention: This is a preview API available since 7.0.0-alpha.2. This API is not in stable state yet. It may change in the future.

</Note>

</PlatformSection>
```